ketcher-autotests/tests/Templates/Template-Manipulations/Template-manipulations.spec.ts

```diff
@@ -0,0 +1,319 @@
+/* eslint-disable no-magic-numbers */
```

Comment:
Could you, please, remove it and ignore for some specific line of code?

Reply:
Done.